tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -84,6 +84,10 @@ setup() {
 }
 
 teardown() {
+	# The last test set the policy to deny all resources,
+	# so we need to set it to allow all resources again for next test
+	# (e.g k8s-sealed-secrets.bats)
+	kbs_set_allow_all_resources
```

Comment:
Ideally we don't want to allow all resources at the end of the test. We want to set the kbs policy back to [default](https://github.com/confidential-containers/trustee/blob/main/kbs/src/policy_engine/opa/default_policy.rego), which is a little bit different.

Reply:
Thanks for feedback. I will update the PR to use the default policy after the current test round is finished. 👍